modules/config.js

```diff
@@ -19,5 +20,24 @@ module.exports = {
 
   setMatchMedia (nextMatchMediaFunction: Function) {
     _matchMediaFunction = nextMatchMediaFunction;
+  },
+
+  setMediaQueryAlias (alias: string, query: string) {
+    if (alias.indexOf('@') !== 0) {
```

Comment:
I wonder if we should make checks like this for development only using `if (__DEV__) { ... }` and some kind of webpack rule.